cluster-setup/ci-cluster/feature_sriov/11-sriov-networknodepolicy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-network-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: false
+  mtu: 1500
+  nicSelector:
+    pfNames:
```

Comment:
As I wrote in the comment, I would put this under base and then kustomize the nic vendor.